src/test/java/com/google/firebase/FirebaseAppTest.java

```diff
@@ -419,6 +423,104 @@ public void testTokenRefresherStateMachine() {
     assertEquals(0, refresher.cancelCalls);
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void testDefaultAppEmptyFile() {
+    setFirebaseConfigEnvironmentVaraible("firebase_config_empty.json");
+    FirebaseApp.initializeApp();
+  }  
+  
+  @Test(expected = NullPointerException.class)
+  public void testDefaultAppEmptyJSONString() {
+    setFirebaseConfigEnvironmentVaraible("");
```

Comment:
This is triggering the NullPointerException. Code below will not get executed.

Reply:
thanks. missed this.